stories/Image.stories.mdx

```diff
@@ -201,39 +275,6 @@ import { Image } from '@wpmedia/engine-theme-sdk';
   </Story>
 </Canvas>
 
-### **Without resizer URL**
```

Comment:
can you update this, to say, Without resizer url, this shouldn't render

Reply:
I don't think in the ticket it says to delete it

Reply:
A couple of the example images do not load – is that intentional? Does this component not work without these params – if so, should we say in the documentation that they’re required? 

Without resizer URL 

No resized params

Reply:
Why have something that doesn't render?  I say in the docs that required params missing will result in a broken image

Reply:
anyways, I oput those two non-working test cases back in with your comment